owls.properties.tmpl

```diff
@@ -40,6 +40,10 @@ openwifi.kafka.enable = ${KAFKA_ENABLE}
 openwifi.kafka.brokerlist = ${KAFKA_BROKERLIST}
 openwifi.kafka.auto.commit = false
 openwifi.kafka.queue.buffering.max.ms = 50
+openwifi.kafka.ssl.ca.location =
```

Comment:
Let's also add config variables here and add them to `docker-entrypoint.sh` with a default empty value.

Reply:
Good notice, changed it.